lib/templates/contact_us.rb

```diff
@@ -27,4 +27,6 @@
   # Configure the redirect URL after a successful submission
   config.success_redirect = '/'
 
+  config.parent_mailer = 'ActionMailer::Base'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.